packages/skill-lesson/trpc/routers/module-progress.ts

```diff
@@ -38,12 +39,20 @@ export const moduleProgressRouter = router({
         },
       })
 
+      const sortedModuleLessonProgress =
+        sortByLessonSlugNotNull(moduleLessonProgress)
+
+      const filteredModuleLessonProgress = uniqBy(
+        sortedModuleLessonProgress,
+        'lessonId',
+      )
```

Comment:
Why do you sort by `lessonSlug` before filtering out records by unique `lessonId`?

I'm both curious and think it would be worth including a comment.

Reply:
you're correct that is not required. I updated it so that it just sorts by `updatedAt` which will be enough to shift the most recent duplicate to the top resulting in `uniq` filtering out the second one.